packages/notifi-react-card/lib/components/intercom/SendMessageSection.tsx

```diff
@@ -4,16 +4,23 @@ import React, { useState } from 'react';
 export type SendMessageSectionProps = Readonly<{
   classNames?: Readonly<{
     container: string;
-    textArea: string;
+    textarea: string;
     button: string;
   }>;
+  sendConversationMessages: (message: string) => void;
 }>;
 
 export const SendMessageSection: React.FC<SendMessageSectionProps> = ({
   classNames,
+  sendConversationMessages,
 }) => {
   const [sendMessage, setSendMessage] = useState<string>('');
```

Comment:
Could we update the initial state to `null` instead? 

The sendConversationMessage function will fire if the message is `''`. We do disable the Send button if the message is `''` but it may be good extra precautions to default `sendMessage` to `null`.

Reply:
Could you explain why the default value null for sendMessage is better than empty string, yeah, as you mentioned, we have made the condition that we disabled the button when it's an empty string. And if we made the default value to null, the type for sendMessage should be string | null instead of just string.

Reply:
Yes, I think overall, we need to have a better pre-cautions to prevent accidental sending of an empty message. In addition, to making the send button disabled, we should also enforce it in the function as well.

I do think the `null` message is better because it is more explicit than an empty string.